obal/data/roles/build_srpm/tasks/main.yaml

```diff
@@ -0,0 +1,14 @@
+---
+- name: 'Build SRPM'
+  command: "tito build --srpm --offline"
```

Comment:
It would be nice if we had a way to build SRPMs for nightly packages as well.

Reply:
It does sound nice, but that seems out of scope for this pull request.